src/pages/sitemap/sitemap-index.xml.ts

```diff
@@ -42,10 +42,17 @@ export const getServerSideProps: GetServerSideProps = async context => {
         priority: 0.5,
     }));
 
+    const academiaPage = {
+        loc: 'https://www.mongodb.com/academia',
+        lastMod: curDate,
+        priority: 1,
```

Comment:
Waiting for Elizabeth from SEO team to confirm what the priority should be for this page